### DISCUSSION TITLE
Should we release 0.67.3?

### DISCUSSION THREAD

**Discussion by ShikaSD** [reactwg/react-native-releases#14] [Patches]
Current Release: [0.67.2](https://github.com/facebook/react-native/releases/tag/v0.67.2)

Conversations on this thread are limited: 
* [major release issues](https://github.com/facebook/react-native/wiki/Release-FAQ#what-is-release-blocking)
* [qualified cherry-pick requests](https://github.com/facebook/react-native/wiki/Release-FAQ#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://github.com/facebook/react-native/wiki/Release-FAQ#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue. 
If the issue is a [major release issue](https://github.com/facebook/react-native/wiki/Release-FAQ#what-is-release-blocking), please reference the issue here.

---
#### List of qualified picks
 
1. https://github.com/facebook/react-native/commit/1a83dc36ce0af33ac7a3c311354fce4bfa5ba1a3
2. https://github.com/facebook/react-native/commit/c1db41f060908e6ab001aaace7c20c610056f59a

#### Local commits to backport to main
1.
```

**Comment by billnbell** [comment]
upgrade helper is not there for 0.67.2 -- ??

**Reply by kelset** [reply]
Thanks for reporting! I've generated it now, if you open the upgrade-helper now you should see 0.67.2 there :)

(I think I might have an idea of why it wasn't automatically generated, I'll look more into it...)

**Comment by kmsbernard** [comment]
I suggest to cherry-pick https://github.com/facebook/react-native/commit/1a83dc36ce0af33ac7a3c311354fce4bfa5ba1a3, fix a broken text input in Korean language on iOS 15

**Comment by lunaleaps** [comment]
Pick request: https://github.com/facebook/react-native/commit/c1db41f060908e6ab001aaace7c20c610056f59a
- Fixes the infinite loop explained in the issue https://github.com/facebook/react-native/issues/33129 by reverting commit  https://github.com/facebook/react-native/commit/59021521e7aba0f70b91b5c7778ccdd1b30eaae4. PR https://github.com/facebook/react-native/pull/31538.

**Comment by kelset** [comment]
(note: if we do 0.67.3 we need to mirror the cherry picks in 0.68 branch to ensure sync)

**Reply by joe-sam** [reply]
It would be even better if this regression prevention can be scripted.
 However I think that major merge conflicts could possibly emerge on an auto cherry pick for the future version compatibility especially if there are drastic version changes and it could prove nearly impossible to do that sort of rebase.
 Still it may be worth a shot.

**Comment by astragofore** [comment]
My team (and judging from many other reactions ([here](https://github.com/react-native-community/releases/issues/245#issuecomment-902195984)) many would appreciate this released: https://github.com/facebook/react-native/pull/31865 
In order to make tests in many applications better functional. In short: resource-id is not available for TextInputs

**Reply by ShikaSD** [reply]
I think it is already in 0.67, looking at the commit :)

**Reply by kelset** [reply]
exactly - in the commit (https://github.com/facebook/react-native/commit/efb359f3184ec26a21044b6145586cf1f5afc438) you can see that it's been part of every RN version >= 0.66:
<img width="1247" alt="Screenshot 2022-03-16 at 10 43 51" src="https://user-images.githubusercontent.com/16104054/158572745-23ea53fa-ba68-4b3c-bafd-92d4f06f5201.png">

unless it was reverted at some point - but as far as I can see in main the changes are still present so it doesn't look like it was ever reverted (example: https://github.com/facebook/react-native/blob/main/ReactAndroid/src/main/java/com/facebook/react/views/scroll/ReactScrollView.java#L131)

**Reply by astragofore** [reply]
excellent, thank you both!
